FSDigitaltechnik.tex

```diff
@@ -603,7 +609,7 @@ \subsection{Deterministic finite state machine (DFA)}
 $g$ & Ausgaberelation \\
 \end{tabular}
 \subsection{Moore und Mealy FSMs}
-\includegraphics{img/ds/automaten.pdf}\\
+\includegraphics{img/ds/automaten_neu.pdf}\\
```

Comment:
Let's replace the existing file if it is not used anymore. We have version control to track the diff

Reply:
Ok, I'll remove `automaten3.pdf` aswell then.